src/modules/consumer.js

```diff
@@ -190,10 +190,10 @@ class Consumer {
         params: { queue, message: messageString },
       });
 
-      // main answer management chaining
-      // receive message, parse it, execute callback, check if should answer, ack/reject message
-      const body = parsers.in(msg);
+      let body = {};
       try {
+        body = parsers.in(msg);
```

Comment:
Any reason you don't user const here. Why the let before the `try/catch`?

```suggestion
        const body = parsers.in(msg);
```